app/components/TuneCard/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ *
+ * TuneCard
+ *
+ */
+
+import React, { memo, useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Typography, Image, Button } from 'antd';
+import { PlayCircleFilled, PauseCircleFilled } from '@ant-design/icons';
+import If from '@components/If';
+
+const ItemCard = styled(Card)`
+  && {
+    margin: 20px 0;
+    display: flex;
+    width: ${(props) => props.maxwidth}px;
+    flex-direction: column;
+    justify-content: space-around;
+  }
+`;
+
+const iconStyle = { fontSize: 32 };
+
+const IconButton = styled(Button)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export function TuneCard({ maxwidth, artistName, collectionName, cardImg, previewUrl, handleOnActionClick }) {
+  const audioRef = useRef();
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  useEffect(() => {
```

Comment:
this is very inefficient, even if its false your constantly setting it as false. Do we really need to do this? 

we can completely remove the need of the `isPlaying` state right?

Reply:
Remove the state from TuneCard